policyengine_us/variables/gov/states/nc/ncdhhs/scca/nc_scca_countable_income.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class nc_scca_countable_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    unit = USD
+    definition_period = MONTH
+    label = "North Carolina Subsidized Child Care Assistance program countable income"
+    reference = "https://policies.ncdhhs.gov/wp-content/uploads/FINAL-Chapter-7-Family-definition-and-determining-income-eligibility-08-05-24.pdf#page=11"
+
+    adds = ["snap_earned_income", "snap_unearned_income"]
```

Comment:
make this a list parameter specifying the income sources per the document. for example, SCCA doesn't count SSI, but SNAP does in its unearned income

Reply:
Confirmed with our benefit specialist and moved all the countable income sources to `gross_earned` param.